frontend/src/app/modules/result-selection/components/result-selection-job-group/result-selection-job-group.component.ts

```diff
@@ -19,7 +16,7 @@ import { SharedService } from '../../services/sharedService';
     encapsulation: ViewEncapsulation.None
   })
 
-export class ResultSelectionJobGroupComponent implements OnInit {
+export class ResultSelectionJobGroupComponent {
   @Input() currentChart: string;
   @Input() jobGroupMappings$: Observable<SelectableApplication[]>;  
```

Comment:
```suggestion
jobGroupMappings$: Observable<SelectableApplication[]>;  
```
You don't need this as an input field.